ms2rescore/rescore_core.py

```diff
@@ -523,6 +523,9 @@ def write_pin_files(
         else:
             pep["Proteins"] = pep["peptide"]
 
+    # format peptide sequence to work with fido TODO: check!!
```

Comment:
To do

I suspect this is to check if the peptide would already be formatted with underscores?